app/src/main/java/kr/co/connect/boostcamp/livewhere/repository/MapRepository.kt

```diff
@@ -0,0 +1,14 @@
+package kr.co.connect.boostcamp.livewhere.repository
+
+import io.reactivex.Single
+import kr.co.connect.boostcamp.livewhere.model.HouseResponse
+import kr.co.connect.boostcamp.livewhere.model.PlaceResponse
+import kr.co.connect.boostcamp.livewhere.model.ReverseGeo
+import retrofit2.Response
+
+interface MapRepository {
+    fun getPlace(lat: Any, lng: Any, radius: Any, category: Any): Single<Response<PlaceResponse>>
```

Comment:
```suggestion
    fun getPlace(lat: String, lng: String, radius: String, category: String): Single<Response<PlaceResponse>>
```
P2. Any 대신 String을 써도 되지 않을까요?